app-server/app/src/test/scala/auth/AuthSpec.scala

```diff
@@ -0,0 +1,76 @@
+package com.azavea.rf.auth
+
+import org.scalatest.{Matchers, WordSpec}
+import akka.http.scaladsl.testkit.{ScalatestRouteTest, RouteTestTimeout}
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.model.headers.{RawHeader, Authorization, OAuth2BearerToken}
+import akka.http.scaladsl.model.{HttpEntity, ContentTypes}
+import akka.actor.ActorSystem
+import concurrent.duration._
+import spray.json._
+import de.choffmeister.auth.common.JsonWebToken
+import akka.http.scaladsl.server.Route
+
+import com.azavea.rf.utils.Config
+import com.azavea.rf.{DBSpec, Router}
+import com.azavea.rf.datamodel.latest.schema.tables.OrganizationsRow
+import com.azavea.rf.utils.PaginatedResponse
+
+class AuthSpec extends WordSpec
+    with Matchers
+    with ScalatestRouteTest
+    with Config
+    with Router
+    with DBSpec {
+  implicit val ec = system.dispatcher
+  implicit def database = db
+  implicit def default(implicit system: ActorSystem) = RouteTestTimeout(DurationInt(20).second)
+  val now = new java.util.Date()
+  val cal = java.util.Calendar.getInstance();
+  cal.setTime(now);
+  cal.add(java.util.Calendar.DATE, 1)
+  val tomorrow = cal.getTime()
+  val claims = Map(
+    "sub" -> JsString("NewUser")
+  )
+  val token = JsonWebToken(claims, now.toInstant(), tomorrow.toInstant())
+  val encodedToken = JsonWebToken.write(token: JsonWebToken, auth0Secret)
+  val authorization = Authorization(OAuth2BearerToken(encodedToken))
+
+  def testRoute: Route = {
```

Comment:
Some comments around these test Routes would be good -- took me a bit to figure out what they were being used for exactly -- renaming them would help as well.